core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1166,6 +1166,27 @@ class WorkspaceService(protected val ctx: RawlsRequestContext,
         fastPassServiceConstructor(childContext, dataSource)
           .syncFastPassesForUserInWorkspace(destWorkspaceContext)
       )
+
+      _ <- Future {
```

Comment:
Why is the Future needed here?

Reply:
Cloning the workspace will take some time, so wrapping it in a Future prevents it from blocking the thread